spec/pipelines/pacbio/message_spec.rb

```diff
@@ -16,17 +17,16 @@
   end
 
   describe 'key' do
-
     let(:key) { message.content[pacbio_config.key] }
 
-    let(:timestamp) { Time.parse('Mon, 08 Apr 2019 09:15:11 UTC +00:00') }
+    let(:timestamp) { Time.zone.parse('Mon, 08 Apr 2019 09:15:11 UTC +00:00') }
 
     before(:each) do
       allow(Time).to receive(:current).and_return timestamp
     end
 
     it 'must have a id_pac_bio_run_lims' do
-      expect(key[:id_pac_bio_run_lims]).to eq(plate.run.id)
+      expect(key[:id_pac_bio_run_lims]).to eq(plate.run.traction_id)
```

Comment:
This was the change I made to let tests pass -- everything else was satisfying Rubocop during the commit